hedyorg/hedy
# 🪲 Remove button for submitting feedback

Removes the button and modal for giving feedback.

Fixes #5988

**How to test**
- Login as a teacher and check that there is no button for providing feedback.
- Login as a super teacher and check that there is no button for providing feedback. However, the super teacher should still be able to see and filter existing feedback.
